packages/vuetify/src/util/helpers.ts

```diff
@@ -415,3 +414,27 @@ export function getUid () {
   return getUid._uid++
 }
 getUid._uid = 0
+
+export function flattenFragments (nodes: VNode[]): VNode[] {
```

Comment:
Didn't end up using this because I decided to pass all the `<source>` elements straight through instead of parsing them, might be useful for something else though.